demos-europe/demosplan-core
# Fix test core permissions

- Adjust permissions test
- Compare core permission vs test and adjust permissions in test

### How to review/test
- Run in local \Tests\Core\Core\Unit\Logic\PermissionsTest
- Test should work

Delete the checkbox if it doesn't apply/isn't necessary.

- [ x] Tests updated/created
